README.md

```
@@ -32,6 +32,61 @@ pnpm dev
+
+`app.config.ts`가 빌드 시점에 이 값을 읽어 네이티브 설정(URL 스킴, `strings.xml`, `Info.plist`)에
+주입합니다. 값이 없으면 prebuild가 **즉시 에러와 함께 멈춥니다.** (키 없이 빌드하면 빌드는 성공하는데
+로그인만 조용히 실패해서, 일부러 먼저 터지게 해뒀습니다.)
+
+> **EAS 클라우드 빌드는 `.env`를 읽지 못합니다.** gitignore 대상이라 업로드되지 않기 때문입니다.
+> 클라우드 빌드를 쓴다면 한 번만 등록해두세요.
+>
+> ```bash
+> eas env:create --name KAKAO_NATIVE_APP_KEY --value 발급받은_네이티브_앱_키
+> ```
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)